src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        Color[] colors = Color.values();
+        return colors[random.nextInt(colors.length)].name();
```

Comment:
NOT MISTAKE, but make a note: You realized method `getName()` in `enum Color` and you can use it for better colors' names.

Reply:
I guess I created it and used it in previous versions, but after that, I read the last point of "Common mistakes" and replaced it by just calling the name method.

Reply:
Fixed. I thought that we needed a new generator to start a new sequence of pseudorandom numbers for each new Ball object, maybe I was confused with another case.